CONTRIBUTING.md

```diff
@@ -173,15 +173,20 @@ C source modules (`.c`) should be organized as follows:
 
 In the rare case that a function declaration and definition are both in different header files (rather than the definition in a `*.c` file) this can cause rendering errors for `Doxygen`. The work around for these cases can be demonstrated with this example for the function `sys_listener_destroy` it is documented and declared in `sys.h` and defined as follows in `listeners.h`:
 
+```c
     #ifndef DOXYGEN_SKIP_SECTION /* documented in sys.h */
     void sys_listener_destroy(struct ListHead *item)
     {
         EventListener *listener = GET_LIST_ENTRY(item, EventListener, listeners_list_head);
         free(listener);
     }
     #endif /* DOXYGEN_SKIP_SECTION */
+```
 
-> Note: You should include a short `/* comment */` trailing the `#ifndef` entry mentioning the file where the function is actually documented.
+```{admonition} Note:
```

Comment:
Is `{admonition}` something parsed from GitHub?

Reply:
I just discovered that this format doesn't work well, so I am already in the process of changing all of the `{admonition}`'s to a more portable style.

Reply:
Done.